plugins/woocommerce-blocks/packages/components/store-notices-container/index.tsx

```diff
@@ -85,20 +85,23 @@ const StoreNoticesContainer = ( {
 	if ( suppressNotices ) {
 		return null;
 	}
+	const snackbarNotices: NoticeType[] = [];
+	const storeNotices: NoticeType[] = [];
+
+	notices.forEach( ( notice ) => {
+		if ( notice.type === 'snackbar' ) {
+			snackbarNotices.push( notice );
+		} else {
+			storeNotices.push( notice );
+		}
+	} );
 
 	return (
 		<>
-			<StoreNotices
-				className={ className }
-				notices={ notices.filter(
-					( notice ) => notice.type === 'default'
-				) }
-			/>
+			<StoreNotices className={ className } notices={ storeNotices } />
 			<SnackbarNotices
 				className={ className }
-				notices={ notices.filter(
-					( notice ) => notice.type === 'snackbar'
-				) }
+				notices={ snackbarNotices }
```

Comment:
To make sure it displays the error notices. 
For express checkout notices, the notice type was neither default nor snackbar, so notices were not getting displayed. 

@opr This is why error notices of express checkout were not visible during our pairing programming session.

Reply:
I would suggest trying to understand why the notice.type is not set since it looks like the [type should be `default` by default](https://github.com/WordPress/gutenberg/blob/a0cb09b99624d30ffff26dee7160292ab505655d/packages/notices/src/store/actions.js#L69-L81) when a notice is created. I'd also want to know _what_ the type is. Perhaps it's being intentionally set to something different than "default" and that's why it's not being displayed because the current implementation only considers whether a notice is one of two types. There are two potential paths as a result of that investigation:

- Explicitly expect and handle whether a notice is a snackbar type or not. In which case, you can simplify things by changing the filter callback condition in the original code.
- Discover that there is a specific type being passed in for a created notice that isn't handled yet and whether that's something that _should_ be handled. 

I'm assuming the first option is the one we'd probably care about here.